src/utils/constants.ts

```diff
@@ -0,0 +1,12 @@
+import { Asset, DDO } from '@oceanprotocol/lib'
+
+export type AssetSpecificProps = Exclude<keyof Asset, keyof DDO>
+
+// these are the props which extend the DDO interface to be an aquarius Asset
+// this const is needed to strip these away from an Asset to convert it into an DDO
+export const AQUARIUS_ASSET_EXTENDED_DDO_PROPS: AssetSpecificProps[] = [
```

Comment:
comment: let's add a TODO to look into type optimization at some point. Not a high prio for this PR imo